### PR TITLE
[HttpKernel] Keep 3.0 compat by not using ContainerAware

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/ContainerAwareTrait.php [hunk 1]
```diff
@@ -24,9 +24,9 @@
     protected $container;
 
     /**
-     * Sets the Container associated with this Controller.
+     * Sets the Container.
```
Comment (Tobion) [inline src/Symfony/Component/DependencyInjection/ContainerAwareTrait.php]: can you lowercase `container` please as it doesnt mean a class but a concept.

Comment (Tobion) [inline src/Symfony/Component/DependencyInjection/ContainerAwareTrait.php]: and these changes should be done in 2.7

File: src/Symfony/Component/DependencyInjection/ContainerAwareTrait.php [hunk 2]
```diff
@@ -24,9 +24,9 @@
     protected $container;
```
Comment (Tobion) [inline src/Symfony/Component/DependencyInjection/ContainerAwareTrait.php]: phpdoc would need to be `ContainerInterface|null` as well

**Comment by nicolas-grekas** [comment]
Comments addressed

**Comment by Tobion** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**